paddle/fluid/inference/tests/api/analyzer_seq_pool1_tester.cc

```diff
@@ -121,14 +121,6 @@ void PrepareInputs(std::vector<PaddleTensor> *input_slots, DataRecord *data) {
   }
 }
 
```

Comment:
please add `compare_determine` UT and test 10000 or 100000 times at local machine to prevent random inference result.

Reply:
OK, done.